src/components/content/MessageWithEmphasis.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    messageID: string;
+    // EmphasisComponent: any;
+}
+
+function MessageWithEmphasis({ messageID }: Props) {
```

Comment:
I noticed this pattern was already started elsewhere and made a component. Did not switch over all the text that followed this pattern.